TheJumpCloud/support
# Use production location for remote assist agent in the commands gallery

## Issues
* [TX-497](https://jumpcloud.atlassian.net/browse/TX-497) - Use production location for remote assist agent in the commands gallery

## What does this solve?
Customers who use the commands gallery to install the remote assist agent get the same version of the binary that is released via slowroll 

## Is there anything particularly tricky?
No

## How should this be tested?
Verify the commands work and install the jumpcloud-remote-assist binaries from `https://jumpcloud-windows-agent.s3.amazonaws.com/production/`

## Screenshots
